build/dockerfiles/event_watcher.Dockerfile

```diff
@@ -15,12 +15,27 @@ RUN go mod download -x
 # Build event_watcher
 FROM base as builder
 
+RUN mkdir -p /scroll/lib/
+RUN apt-get -qq update && apt-get -qq install -y wget
+RUN wget -O /scroll/lib/libzktrie.so https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libzktrie.so
+RUN wget -O /scroll/lib/libscroll_zstd.so https://github.com/scroll-tech/da-codec/releases/download/v0.0.0-rc0-ubuntu20.04/libscroll_zstd.so
```

Comment:
in the future we can refactor and make the <libscroll_version> an arg. see https://github.com/scroll-tech/scroll/blob/0be158e27df1221a1c62845d7881dcf02dc11e7a/build/dockerfiles/intermediate/go-rust-builder.Dockerfile#L28

Reply:
make sense.

Reply:
changed in [b215558](https://github.com/scroll-tech/scroll/pull/1298/commits/b215558a423efda04c2edeadbd7e518f4fe8190c).